controller/volume_controller.go

```diff
@@ -4434,7 +4439,7 @@ func (c *VolumeController) createSnapshot(snapshotName string, labels map[string
 		}
 	}
 
-	snapshotName, err = engineClientProxy.SnapshotCreate(e, snapshotName, labels)
+	snapshotName, err = engineClientProxy.SnapshotCreate(e, snapshotName, labels, freezeFilesystem)
```

Comment:
Looks good to me because looks like this function is used for volume cloning so we still need to freeze filesystem